presto-client/src/main/java/io/prestosql/client/QueryError.java

```diff
@@ -76,6 +79,13 @@ public String getErrorName()
         return errorName;
     }
 
+    @Nullable
```

Comment:
Let's use `Optional` instead. Also I would use `Optional<String>` as type for `semanticErrorName` field as well.